### DISCUSSION TITLE
Thumbnail generation using ghostscript+imagemagick in the .NET application.

### DISCUSSION THREAD

**Discussion by Vicchustreakz** [ImageMagick/ImageMagick#7124] [Help]
### ImageMagick version

 7.1.0-49

### Operating system, version and so on

Windows 11

### Description

I have added ghostscript to the project and while checking the version of the imagemagick it listed the supported delegates and it showed this 
Delegates (built-in): bzlib cairo freetype gslib heic jng jp2 jpeg jxl lcms lqr lzma openexr pangocairo png ps raqm raw rsvg tiff webp xml zip zlib

since it shows gslib. I have added the ghostscript application in my .NET project. I am using Imagemagick as a standalone application in my project folder and ghostscript in another folder. And I have also placed these applications inside the Debug folder of the project so once I build the package it'll be useful. So from the forums and Chat GPT it says " ImageMagick should automatically utilize Ghostscript for EPS support when converting EPS files to other formats like JPG." 

How do I ensure that .NET project is configured to use ImageMagick with Ghostscript support for EPS conversion.

**Comment by dlemstra** [comment]
You can make sure this happens by installing Ghostscript on your machine or copy the binaries to your project and then use `MagickNET.SetGhostscriptDirectory(@"c:\your\path")` to specify the path.
